pkg/scheduler/core/generic_scheduler_test.go

```diff
@@ -66,6 +66,8 @@ var (
 	errPrioritize = fmt.Errorf("priority map encounters an error")
 )
 
+const ErrFakePredicate = "Nodes failed the fake predicate"
```

Comment:
Name it ErrReasonFake, predicates are deprecated

Reply:
Done  thx~